### PR TITLE
Fixed: Confirmation Modal not opening

### PR DESCRIPTION
**Pull request by vinaybadgujar102**

### CODE REVIEW

**Comment by vinaybadgujar102** [comment]
#1216

File: src/common/ConfirmationModal.tsx [hunk 1]
```diff
@@ -53,19 +52,19 @@ const ConfirmationModal: React.FC<ConfirmationModalProps> = ({ action, handleCli
       className={`popupModal${darkModeStatus ? "-dark" : ""} ${darkModeStatus ? "dark" : "light"}-theme${theme[darkModeStatus ? "dark" : "light"]}`}
     >
 
-      <h5>{header}</h5>
-      <p>Note: {note}</p>
+      <h5>{t(headerKey)}</h5>
```
Comment (Tushar-4781) [inline src/common/ConfirmationModal.tsx:55]: Please use <p> tag here and use class "popupModal-title" with inline style margin = 0

**Comment by Tushar-4781** [comment]
@vinaybadgujar102 now you have created a PR please continue

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/common/ConfirmationModal.tsx (hunk 1 above)
Comment (vinaybadgujar102) [reply src/common/ConfirmationModal.tsx]: made the changes

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/common/ConfirmationModal.tsx [hunk 2]
```diff
@@ -53,19 +52,19 @@ const ConfirmationModal: React.FC<ConfirmationModalProps> = ({ action, handleCli
...
           className="checkbox"
           onChange={() => { setNeverShowAgain(!neverShowAgain); }}
-        > Don&apos;t ask again for this action?
+        > {t("dontAskAgain")}
         </Checkbox>
       </div>
       <div style={{ display: "flex", justifyContent: "space-around" }}>
-        { getChoiceButton(getConfirmButtonText(actionName)) }
-        { getChoiceButton("Cancel") }
+        { getChoiceButton(t(getConfirmButtonText(actionName))) }
```
[the remark this answers is not in the document]
Comment (vinaybadgujar102) [reply src/common/ConfirmationModal.tsx:67]: check if the approach is correct now. Added dynamic translation to "getConfirmTextButton" object which helps in matching the conditional checks even in different languages.

**Comment by Tushar-4781** [comment]
@vinaybadgujar102 any reason behind marking this PR as draft ?

**Comment by vinaybadgujar102** [comment]
> @vinaybadgujar102 any reason behind marking this PR as draft ?

I thought it might take some time. So changed to draft.

**Review by Tushar-4781 — APPROVED** [review]
The primary scope of this PR is resolved

**Pull request merged**